in-relentless-pursuit-of-rest.md

```diff
@@ -0,0 +1,31 @@
+# In Relentless Pursuit of REST
+
+## Abstract
+
+"That's not very RESTful." As a Rails developer you've probably heard or even
+spoken that proclamation before, but what does it really mean? What's so great
+about being RESTful anyway?
+
+In this talk, you'll learn to refactor existing code to follow RESTful
+principles. Together we'll discuss the changes and the clarity of purpose they
```

Comment:
What do you think about re-ordering the first two sentences of this? It feels like we jumped from "what's so great about being restful?" to "clearly you want to be restful and we'll teach you how", while the (first half of the) second sentence helps transition that.